common/util/bbb.ts

```diff
@@ -184,29 +164,21 @@ function mapJSONtoAttendee(o: any): Attendee {
                         o && o.role && o.role.length > 0 && o.role[0]);
 }
 
-function lessThanDate(date1: Date, date2: Date): boolean {
-    if (date1.getFullYear() < date2.getFullYear()) {
-        return true;
-    } else if (date1.getMonth() < date2.getMonth()) {
-        return true;
-    } else if (date1.getDate() < date2.getDate()) {
-        return true;
-    }
-    return false;
-}
-
 // Returns active lecture of the subcourse, assuming that there is only one active lecture of the subcourse
 async function getActiveLectureOfSubcourse(subcourseId: string): Promise<Lecture> {
     const entityManager = getManager();
     const lectures = await entityManager
         .createQueryBuilder(Lecture, "lecture")
         .where("lecture.subcourse.id = :id", {id: subcourseId})
         .getMany();
-
+    const now = new Date();
     // check if lecture is running now (lecture.start + duration > now)
     for (const lecture of lectures) {
-        if (!lessThanDate(lecture.start, new Date())
-            && (lecture.start.getTime() + (lecture.duration * 60000)) > new Date().getTime()) {
+
+        if (lecture.start.getFullYear() == now.getFullYear() &&
+            lecture.start.getMonth() == now.getMonth() &&
+            lecture.start.getDate() == now .getDate() &&
+            (lecture.start.getTime() + (lecture.duration * 60000)) > new Date().getTime()) {
```

Comment:
Looks good, but we could also use `now` here instead of `new Date()`, as in the lines above.

Reply:
You're right.